.browserslistrc

```diff
@@ -15,3 +15,5 @@ not dead
 not IE 9-11 # For IE 9-11 support, remove 'not'.
 not op_mob <= 73
 not android <= 109
+not kaios 2.5
+not op_mini all
```

Comment:
What are these doing?  When I start the app I get:

```
One or more browsers which are configured in the project's Browserslist configuration will be ignored as ES5 output is not supported by the Angular CLI.
Ignored browsers: kaios 2.5, op_mini all
```

Reply:
Those changes should get rid of those errors which is why they are included.